wlutil/build.py

```diff
@@ -186,23 +186,78 @@ def buildWorkload(cfgName, cfgs, buildBin=True, buildImg=True):
     if ret != 0:
         raise RuntimeError("Error while building workload")
 
+def makeDrivers(boardDir, linuxSrc):
+    driverDirs = pathlib.Path(boardDir).glob("drivers/*")
+    makeCmd = "make -C " + str(linuxSrc) + " ARCH=riscv CROSS_COMPILE=riscv64-unknown-linux-gnu- "
```

Comment:
I do my own manual make command here instead of using the makefiles included with the drivers themselves. On the one hand, it's a bit safer, but on the other, it means any custom things done down the line in those Makefiles wont propagate to Marshal. Not sure what the best option is.

Reply:
After discussion, this has been switched to just all the driver's own makefile